src/authentication/login.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsString } from "class-validator"
+
+class LogInDto {
+    @IsString()
+    public email: string
+
+    @IsString()
```

Comment:
```suggestion
    @IsString()
    @MinLength(8)
    @MaxLength(32)
    @IsString()
    @Matches(/^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#%^&*])/, {message: "Password is too weak. It must contain at least 1 lowercase letter, at least 1 uppercase leter, at least 1 numeric character and at least one special character from: !@#$%^&*"})
```

Walidacja długości oraz siły hasła

Reply:
Done.